demos/python_demos/object_detection_demo_yolov3_async/object_detection_demo_yolov3_async.py

```diff
@@ -161,44 +161,35 @@ def parse_yolo_region(blob, resized_image_shape, original_im_shape, params, thre
     orig_im_h, orig_im_w = original_im_shape
     resized_image_h, resized_image_w = resized_image_shape
     objects = list()
-    predictions = blob.flatten()
-    side_square = params.side * params.side
     size_normalizer = (resized_image_w, resized_image_h) if params.isYoloV3 else (params.side, params.side)
+    bbox_size = params.coords + 1 + params.classes
     # ------------------------------------------- Parsing YOLO Region output -------------------------------------------
-    for i in range(side_square):
-        row = i // params.side
-        col = i % params.side
-        for n in range(params.num):
-            obj_index = entry_index(params.side, params.coords, params.classes, n * side_square + i, params.coords)
```

Comment:
Looks like the `entry_index` function could now be deleted.

Reply:
Thanks